ivy/functional/frontends/jax/devicearray.py

```diff
@@ -16,3 +38,75 @@ def reshape(self, new_sizes, dimensions=None):
 
     def add(self, other):
         return jax_frontend.add(self.data, other)
+
+    @_frontend_wrapper
+    def __add__(self, other):
+        return jax_frontend.add(self.data, other)
+
+    @_frontend_wrapper
+    def __radd__(self, other):
+        return jax_frontend.add(other, self.data)
+
+    @_frontend_wrapper
+    def __sub__(self, other):
+        return jax_frontend.sub(self.data, other)
+
+    @_frontend_wrapper
+    def __rsub__(self, other):
+        return jax_frontend.sub(other, self.data)
+
+    @_frontend_wrapper
+    def __mul__(self, other):
+        return jax_frontend.mul(self.data, other)
+
+    @_frontend_wrapper
+    def __rmul__(self, other):
+        return jax_frontend.mul(other, self.data)
+
+    @_frontend_wrapper
+    def __div__(self, other):
+        return jax_frontend.div(self.data, other)
+
+    @_frontend_wrapper
+    def __rdiv__(self, other):
+        return jax_frontend.div(other, self.data)
+
+    @_frontend_wrapper
+    def __mod__(self, other):
+        return jax_frontend.rem(self.data, other)
+
+    @_frontend_wrapper
+    def __rmod__(self, other):
+        return jax_frontend.rem(other, self.data)
+
+    @_frontend_wrapper
+    def __divmod__(self, other):
+        return divmod(self.data, other)
```

Comment:
the `divmod` from which module is this using?

Reply:
there's actually a `divmod` todo on the list so would you like to implement it for the special method? thanks!

Reply:
I was actually using the in-built `divmod` but since it's on the todo I'll implement it. Thanks!